### PR TITLE
Add 3 items to COPY_ATTRIBUTES which are used by iOS.

### PR DESCRIPTION
**Pull request by montehurd**
[Background](https://github.com/wikimedia/wikipedia-ios/pull/1971#issuecomment-350352888).

https://phabricator.wikimedia.org/T175323

### CODE REVIEW

File: src/transform/LazyLoadTransform.js [hunk 1]
```diff
@@ -15,7 +15,10 @@ const IMAGE_LOADED_CLASS = 'pagelib_lazy_load_image_loaded' // Download complete
 
 // Attributes copied from images to placeholders via data-* attributes for later restoration. The
 // image's classes and dimensions are also set on the placeholder.
-const COPY_ATTRIBUTES = ['class', 'style', 'src', 'srcset', 'width', 'height', 'alt']
+// The 3 data-* items are used by iOS.
+const COPY_ATTRIBUTES = ['class', 'style', 'src', 'srcset', 'width', 'height', 'alt',
+  'data-file-width', 'data-file-height', 'data-image-gallery'
```
Comment (niedzielski) [inline src/transform/LazyLoadTransform.js]: This seems like a pretty harmless change. It's entirely up to your discretion but you may want to update tests for these new attributes. We currently validate all others and don't have to worry about breaking things so much when a change is made.

Just an aside and not a request for revision: on line 77, these attributes will be copied to `data-data-*` which is a bit clumsy. On the other hand, they can be handled generically without any other changes which is great.

Comment (montehurd) [reply src/transform/LazyLoadTransform.js]: Ya I had to account for the `data-data` prefix when implementing tap handling https://github.com/wikimedia/wikipedia-ios/pull/1971/commits/8d50837b8da954cb3dfeef276f1fcb56b89eb44e on the placeholders (ie if a user taps the placeholder span it was nice to have that still try to show the native gallery for that placeholder's image). Agreed it's a bit clumsy, but figured we could circle back later if we refactor in future.

Comment (montehurd) [reply src/transform/LazyLoadTransform.js]: @niedzielski Tests added :)

**Comment by montehurd** [comment]
@dbrant gave thumbs-up on irc. merge time!

**Pull request merged**